### PR TITLE
HOTFIX - TMEDIA-172 - Advanced Right Rail switch to CSS layout

### PR DESCRIPTION
**Pull request by matthewroach**
## Description

Switch the advanced right rail layout to use a CSS approach

Downside - Unable to maintain tab order across all devices based on the order of elements wanted by the layout.

## Jira Ticket
- [TMEDIA-172](https://arcpublishing.atlassian.net/browse/TMEDIA-172)

## Test Steps

1. Checkout this branch `git checkout HOTFIX-TMEDIA-172-advanced-right-rail`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/right-rail-advanced-block`
3. Validate any pages using the advanced right rail layout continue to work as expected
    * http://localhost/pf/right-rail-advanced/?_website=the-sun

Desktop is 
Main 1.    |   Right Rail Top
Main 2.    |  Right Rail Middle
                |  Right Rail Bottom

Tablet and Lower is
Right Rail Top
Main 1
Right Rail Middle
Main 2 
Right Rail Bottom

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [ ] Confirmed all the test steps a reviewer will follow above are working.
- [ ] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [ ] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [ ] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [ ] Confirmed this PR has unit test files
  - [ ] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.jsx [hunk 1]
```diff
@@ -108,6 +64,10 @@ const RightRailAdvancedLayout = ({ children }) => {
   );
 };
 
+const RenderChild = ({ Item, tabletPlacement }) => (
+  <div className={`advanced-grid-${tabletPlacement} layout-section`}>{Item}</div>
```
Comment (brentmiller1973) [inline blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.jsx:68]: Brilliant!

File: blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.scss [hunk 2]
```diff
@@ -8,37 +8,70 @@ body {
...
+.advanced-grid-desktop-main-area {
+  padding: 0;
+  @media screen and (min-width: map-get($grid-breakpoints, 'lg')) {
+    border-right: 1px solid $border-rule-color;
+    padding: 0 map-get($spacers, 'lg') 0 0;
+  }
+}
+
+
+@media (max-width: 64rem) {
```
Comment (brentmiller1973) [inline blocks/right-rail-advanced-block/layouts/right-rail-advanced/default.scss:43]: Was this tested under IE 11?.....I believe it's still supported.  Need to make sure it falls back gracefully.....

**Review by brentmiller1973 — DISMISSED** [review]
<borat-comment> I like very much </borat-comment>. Great job man

**Comment by matthewroach** [comment]
We are pushing to Beta first not stable - New PR - https://github.com/WPMedia/fusion-news-theme-blocks/pull/791

**Pull request closed without merging**